tests/main/lxd/task.yaml

```diff
@@ -117,6 +117,8 @@ execute: |
         echo "Install snapd in container"
         lxd.lxc exec $cont_name -- /root/prep-snapd-in-lxd.sh
         lxd.lxc exec $cont_name -- snap set system experimental.refresh-app-awareness=$REFRESH_APP_AWARENESS_INNER
+        lxd.lxc file push "$(find "$SPREAD_PATH" -name '*.snap.keep')" "$cont_name"/
```

Comment:
this would be the first test that refers to the .keep files outside of preparation, also those exist only USE_PREBUILT_SNAPD_SNAP afaiu, why do we need to this push? should we take the snapd from another place instead?  if we really need this it needs a comment

Reply:
AFAIK this is the only place that has snapd snap that we are running on the host.

Reply:
This has been addressed. Thanks for the help @maykathm!